model_compression_toolkit/__init__.py

```diff
@@ -27,4 +27,16 @@
 from model_compression_toolkit.common.defaultdict import DefaultDict
 from model_compression_toolkit.common import network_editors as network_editor
 
+import importlib
+
+tf_spec = importlib.util.find_spec("tensorflow")
+was_tf_found = tf_spec is not None
+
+if was_tf_found:
+    from model_compression_toolkit.keras_framework.quantization_facade import keras_post_training_quantization, \
```

Comment:
1. Why change keras to keras_framework ?
2. This verification should be inside the keras_post_training_quantizaiton and if it is called without keras then there is an error message